resources/migrations/000_migrations.yaml

```diff
@@ -13990,6 +14100,62 @@ databaseChangeLog:
             constraintName: fk_action_creator_id
             nullable: false
 
+  - changeSet:
+      id: v46.00-068
+      author: adam-james
+      comment: Added 0.46.0 - Copy activity -> activity_log (most recent 30 days)
+      changes:
+        - sql:
+            dbms: mysql,mariadb,h2
+            sql: >-
+              INSERT INTO activity_log (table_id, database_id, topic, details, model_id, user_id, timestamp, model)
+              SELECT table_id, database_id, topic, details, model_id, user_id, timestamp, model
+              FROM activity WHERE timestamp > now() - INTERVAL '30' day
+              ORDER BY timestamp ASC;
+        - sql:
+            dbms: postgresql
+            sql: >-
+              INSERT INTO activity_log (table_id, database_id, topic, details, model_id, user_id, timestamp, model)
+              SELECT table_id, database_id, topic, details, model_id, user_id, timestamp, model
+              FROM activity WHERE timestamp > (now() - CAST('30 days' AS interval))
+              ORDER BY timestamp ASC;
+      rollback: # we do not care to preserve the activity_log from 46->45
+        - sql:
+            sql: >-
+              DELETE FROM activity_log;
+
+  - changeSet:
+      id: v46.00-069
+      author: adam-james
+      comment: Added 0.46.0 - Copy view_log -> activity_log (most recent 30 days)
+      changes:
+        - sql:
+            dbms: mysql,mariadb,h2
+            sql: >-
+              INSERT INTO activity_log (table_id, database_id, topic, details, model_id, user_id, timestamp, model)
+              SELECT NULL AS table_id,
+                     NULL AS database_id,
+                     CONCAT(model, '-view') AS topic,
+                     metadata AS details,
+                     model_id, user_id, timestamp, model
+              FROM view_log WHERE timestamp > now() - INTERVAL '30' day
+              ORDER BY timestamp ASC;
```

Comment:
Same question as above, does this not work on postgres?

Reply:
Hmm, yea, it does. Not sure why I didn't try that... thanks :)